tests/system_tests_http1_adaptor.py

```diff
@@ -61,6 +61,11 @@
 SERVER_KEY_NO_PASS = RouterTestSslBase.ssl_file('server-private-key-no-pass.pem')
 CHAINED_PEM = RouterTestSslBase.ssl_file('chained.pem')
 
+# Management Entity IDs
+LISTENER_TYPE = 'io.skupper.router.httpListener'
```

Comment:
These constants are already defined in system_test.py which can be imported here.

Reply:
Ok